src/site/_includes/components/header/macro.njk

```diff
@@ -0,0 +1,59 @@
+            ></path>
+          </svg>
+        </span>
+      </a>
+    </div>
+    {% if params.serviceName or params.navigation  %}
+    <div class="govuk-header__content">
+    {% if params.serviceName %}
+    {% if params.serviceUrl %}
+      <a href="{{ params.serviceUrl }}" class="govuk-header__link moduk-header__service-name govuk-header__service-name">
+        {{ params.serviceName }}
+      </a>
+    {% else%}
+      <span class="moduk-header__service-name govuk-header__service-name">
+        {{ params.serviceName }}
+      </span>
+    {% endif %}
+    {% endif %}
+    
+    
+      <button type="button" class="govuk-header__menu-button guidance-js-header-toggle" aria-controls="navigation" aria-label="{{ params.menuButtonLabel | default('Show or hide menu') }}" hidden>{{ menuButtonText }}</button>
+    </div>
```

Comment:
I had the copy the modukHeader and remove the navigation logic from here. The file largely remains the same.